service/history/nDCHistoryReplicator.go

```diff
@@ -149,10 +149,18 @@ func newNDCHistoryReplicator(
 			return newNDCWorkflowResetter(shard, transactionMgr, domainID, workflowID, baseRunID, newContext, newRunID, logger)
 		},
 		newStateBuilder: func(
-			msBuilder mutableState,
+			state mutableState,
```

Comment:
mutableState?

Reply:
cannot name it mutable state since above mutable state is already used